tests/test_fips.py

```diff
@@ -105,20 +105,24 @@ def test_openssl_binary(
     img_id = container_runtime.get_image_id_from_stdout(cmd.stdout)
 
     exec_cmd = " ".join(
-        [container_runtime.runner_binary, "run", "--rm"]
+        [container_runtime.runner_binary, "run", "--rm", "--privileged=true"]
         + get_extra_run_args(pytestconfig)
         + [img_id]
     )
 
     for digest in FIPS_DIGESTS:
-        host.run_expect([0], f"{exec_cmd} /bin/fips-test {digest}")
+        host.run_expect(
+            [0],
+            f"{exec_cmd} bash -c 'mount --bind /tmp/f /proc/sys/crypto; fips-test {digest}'",
+        )
 
     for digest in NONFIPS_DIGESTS:
         err_msg = host.run_expect(
-            [1], f"{exec_cmd} /bin/fips-test {digest}"
+            [1],
+            f"{exec_cmd} bash -c 'mount --bind /tmp/f /proc/sys/crypto; fips-test {digest}'",
```

Comment:
In the CI test run, this returns 0.

Reply:
Now it's a 32 :face_with_spiral_eyes: 
Also: `mount: /proc/sys/crypto: mount point does not exist`

Reply:
I know, apparently the ubunto workers need some kernel module that doesn't autoload.